File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.cs
using System;
using System.Text;

namespace CsTutorial {
    class Program {
        static void Main(string[] args) {
            string[] str = Console.ReadLine().Split();
            int N = int.Parse(str[0]);
            int X = int.Parse(str[1]);
            string[] nums = Console.ReadLine().Split();
            StringBuilder answer = new StringBuilder();
            for (int i = 0; i < N; ++i)
                if(X > int.Parse(nums[i]))
                    answer.Append(nums[i] + " ");
            Console.WriteLine(answer.ToString());
        }
    }
}

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');

const [N, X] = input[0].split(' ').map(Number);
const nums = input[1].split(' ').map(Number);

let answer = [];
for (let i = 0; i < N; ++i)
    if (X > nums[i])
        answer.push(nums[i]);

console.log(answer.join(' '));